test/setup.js

```diff
@@ -0,0 +1,74 @@
+const path = require('path')
+const Application = require('spectron').Application
+const fs = require('fs-extra')
+
+const TEST_DATA_DIR = path.join(__dirname, 'tempTestData')
+
+module.exports = {
+  TEST_DATA_DIR,
+  createApp,
+  endTest,
+  screenshotCreateOrCompare,
+  waitForLoad,
+  wait
+}
+
+// Runs WebTorrent Desktop.
+// Returns a promise that resolves to a Spectron Application once the app has loaded.
+// Takes a Tape test. Makes some basic assertions to verify that the app loaded correctly.
+function createApp (t) {
+  return new Application({
+    path: path.join(__dirname, '..', 'node_modules', '.bin',
+      'electron' + (process.platform === 'win32' ? '.cmd' : '')),
+    args: [path.join(__dirname, '..')],
+    env: {NODE_ENV: 'test'}
+  })
+}
+
+// Starts the app, waits for it to load, returns a promise
+function waitForLoad (app, t) {
+  return app.start().then(function () {
```

Comment:
I added `wipeTestDataDir` to delete everything. Not just saved prefs but all of config.json + torrents + torrent data. Clean slate.